resources/lib/playerinfo.py

```diff
@@ -74,7 +84,7 @@ def onAVStarted(self):  # pylint: disable=invalid-name
         self.quit.clear()
         self.update_position()
         self.update_total()
-        self.push_position()
+        self.push_position(position=self.last_pos, total=self.total)
```

Comment:
I think we need to add a comment here, because it may not be trivial to others why we update the position at the start.

Reply:
Done.